### PR TITLE
[DOC] fix typos

### PR DESCRIPTION
**Pull request by Mouvedia**
## Purpose

fix english typos in the doc files, nothing fancy

### CODE REVIEW

File: media/doc/notes [hunk 1]
```diff
@@ -413,13 +413,13 @@ Something(
 >
 >The aDataIn buffer will not exist after DeviceIoControl returns (and
 >when the I/O operation terminates). I know that for buffered IO the
->input data buffer is copyed by de IOManager to a nonpaged-pool area
+>input data buffer is copied by de IOManager to a nonpaged-pool area
```
Comment (HBelusca) [inline media/doc/notes]: "de IOManager" ;)

File: media/doc/notes [hunk 2]
```diff
@@ -413,13 +413,13 @@ Something(
...
 >before passing the request to driver dispatch routine (DeviceControl).
 >At the point of calling the dispatch routine (DeviceControl) the driver
 >runs in the context of the calling thread so DeviceIoControl hasn't
 >returned yet (?? or so I think) so aDataI
 n will still be valid at the
```
Comment (HBelusca) [inline media/doc/notes]: spurious newline (between "aDataI" and "n will").

File: media/doc/winsta and desktops.txt [hunk 3]
```diff
@@ -94,7 +94,7 @@ DESKTOP:
 be associated with Winsta0
 -Only one desktop is visible to the user and only one can receives input at any time
 -contain a logical display surface
--contin windows (or a pointer to the desktop window)
+-contain windows (or a pointer to the desktop window)
```
Comment (HBelusca) [inline media/doc/winsta and desktops.txt]: btw, shouldn't it be "contains" (every time)?

**Comment by Mouvedia** [comment]
I was just checking your PR process. I can't comply, sorry.

**Comment by Mouvedia** [comment]
reactos/reactos-deprecated-gitsvn-dont-use#21

**Pull request closed without merging**